components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,107 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025, Carsten Grzemba. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# graphical desktop
+depend fmri=mate_install type=require
+
+# DHCP service
+depend fmri=service/network/dhcp/isc-dhcp type=require
+# empty package only for satisfy installation requirements
+# SRSS on Solaris uses the old Sun DHCP server but on OI we use ISC-DHCP
+depend fmri=service/network/dhcp type=require
+
+# SUNWutkio-libs package dependency, empty packages only
+# for satisfy installation requirements
+depend fmri=consolidation/X/X-incorporation type=require
+depend fmri=compatibility/packages/SUNWxwplt type=require
+
+# for Sun JRE
```

Comment:
Why is this needed?

Reply:
SunRay service auth.utauthd.utaut is a Java application. This uses also native libraries from SRSS.